documentation/service/technical/searching-by-location.md

```diff
@@ -0,0 +1,175 @@
+    - [Precomputing data when possible](#precomputing-data-when-possible)
+    - [Tweaking queries to improve performance](#tweaking-queries-to-improve-performance)
+    - [Increasing our database instance resources](#increasing-our-database-instance-resources)
+- [Vacancies location](#vacancies-location)
+- [How do we get the coordinates for the search location](#how-do-we-get-the-coordinates-for-the-search-location)
+- [Location Polygons](#location-polygons)
+  - [Purpose](#purpose)
+  - [Mappings](#mappings)
+- [Getting the geographical coordinates](#getting-the-geographical-coordinates)
+  - [Google Geocoding API costs](#google-geocoding-api-costs)
+
+
+## Introduction
+
+Jobseekers searching for vacancies within a relative distance from a location is a core feature in Teaching Vacancies.
+
+At the time this was written, around 70% of our vacancy searches include location in their filters.
+
+## Our approach
+
+Once a job search location is submitted, the service:
+
+1. Gets the area or coordinates for the searched location.
+
+2. Filters vacancies that, after being included by the rest of the search filters, are located within the provided radius distance from the given location.
+
+3. To allow ordering the search results by distance, computes the exact distance between each vacancy and the given location.
+
+## How we store locations
+
+Our PostgreSQL database instance has the PostGIS extension.
+This extension allows our database to store and operate with geospatial data.
+
+The data type in the database is `geometry` with the `geographic: true` flag and `SRID 4326`.
+
+What does this mean?
+
+We store the data in a geographic coordinate system (GCS) rather than a projected coordinate system (PCS).
+The coordinates are stored as latitude and longitude on the Earth's surface, following the WGS84 standard.
+
+Using the `geometry` type with `geographic: true` allows us to leverage a wide range of PostGIS geospatial functions, such as `ST_Distance`, `ST_Within`, `ST_Intersects`, and `ST_Buffer`.
+
+### Meaning of `SRID 4326`
+
+- SRID: Spatial Referencing System ID.
+- 4326: Represents spatial data using longitude and latitude coordinates on the Earth's surface as defined in the WGS84 standard.
+
+The data type we use from PostGIS is `Geometry` with a `Geographic` coordinate system, which allows us to use projections and transformations.
+
+### Why not just Geometry
+
+When using a geographic coordinate system, distance calculations are performed on the curved surface of the Earth.
+PostGIS functions like `ST_Distance` and `ST_Buffer` will take into account the Earth's curvature, providing more accurate results for large distances compared to a planar (flat) coordinate system.
+
+When investigating the possibility of using Geometry and a Projected Coordinate System, we found that our search area (England, The UK) distances are large enough to need to take the Earth's curvature into account.
```

Comment:
would be good to expand on this, as see y comment above I'm really surprised! (and wondering about journeys involving snake pass!)

Reply:
I originally read that geometry was appropriate for city-scale distances and determined that long distances within "England" were big enough to take the earth's curvature into account. But since the biggest distances we calculate are 200 miles over regions and knowing what we use the areas/distances for doesn't require such a high precision...  we could migrate to Geometry  (unless I miss any PostGIS functionality restriction).

Will write this down and ensure the tech debt ticket points to this context.